zmon_aws_agent/aws.py

```diff
@@ -360,10 +360,13 @@ def get_running_elbs_application(region, acc):
             'elb_type': 'application',
             'dns_name': e['DNSName'],
             'host': e['DNSName'],
-            'name': name,
+            'name': '/'.join(arn.rsplit('/')[-3:]),  # name used by Cloudwatch!
```

Comment:
hmm, I would keep the name as it was actually (the fact that we need to use the last parts for CloudWatch should not have any effect here)